spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
+export PATH="/usr/local/pyload/bin:$PATH"
+
+case $1 in
+    start)
+        if [ -f /usr/syno/etc/ssl/ssl.crt/server.crt -a \
+             -f /usr/syno/etc/ssl/ssl.key/server.key ]; then
+            ln -s /usr/syno/etc/ssl/ssl.crt/server.crt /usr/local/pyload/etc/ssl.crt
+            ln -s /usr/syno/etc/ssl/ssl.key/server.key /usr/local/pyload/etc/ssl.key
+            sed -i -e 's/bool https : "Use HTTPS" = \(True|False\)/bool https : "Use HTTPS" = True/' /usr/local/pyload/etc/pyload.conf
+        else
+            sed -i -e 's/bool https : "Use HTTPS" = \(True|False\)/bool https : "Use HTTPS" = False/' /usr/local/pyload/etc/pyload.conf
+        fi
+        ${PYLOADCORE} --daemon
```

Comment:
I know. I have to look on it in detail because there were struggles about changing ownership and permission of downloaded files.